src/reducers/AuthReducer.js

```diff
@@ -17,13 +18,16 @@ export default (state = INITIAL_STATE, action) => {
     switch (type) {
 
         case APP_READY:
-            return({...state, appReady: true});
+            return ({...state, appReady: true});
+
+        case MAP_READY:
+            return ({...state, appReady: true});
```

Comment:
When `SearchScreen` finishes loading the map, it fires off `appReady: true` for the splash screen to go away. Again, this true/false logic doesn't seem to be triggering the splash screen as I'd expect, but I'll leave the code as-is for now and come back to it. It doesn't seem to be causing any issues.